server/src/main/java/io/crate/protocols/postgres/PostgresWireProtocol.java

```diff
@@ -460,12 +460,12 @@ private void switchToTransportProtocol(Channel channel) {
         addTransportHandler.accept(pipeline);
     }
 
-    private void sendParams(Channel channel) {
+    private void sendParams(Channel channel, Session session) {
```

Comment:
I would rather pass only the `sessionSettings()` not the whole `session`

Reply:
done